README.md

```diff
@@ -1 +1,303 @@
-# edgemesh
\ No newline at end of file
+# EdgeMesh
+
+English | [简体中文](./README_ZH.md)
+
+
+
+## Introduction
+
+EdgeMesh is closely related to KubeEdge, and provides a simple network solution for the inter-communications between services at edge scenarios.
```

Comment:
EdgeMesh is part of KubeEdge

Reply:
I have modified this content as required.